web/src/components/Editor/Expression/grammar/syntax.grammar

```diff
@@ -0,0 +1,41 @@
+@top Program { expression }
+
+@skip { space }
+
+// attr:tracetest.span.type = "${env:TRACETEST_SPAN_TYPE}"
+// att:tracetest.span.duration + 2 = 5 + env:TRACETEST_SPAN_DURATION
+// 5 + 5 = 10
+// 10 != 20
+// "Jorge" == "Ecuador" + ${attr:http.message}
+// att:http.message = "error at line ${attr:tracetest.span.duration}"
+
+expression {Expression Comparator Expression}
+
+Expression {ComposedValue (Operator ComposedValue)*}
+ComposedValue {OutsideInput (Pipe SupportedParsers)* | Number | ComposedString}
+OutsideInput {Source Identifier}
+Interpolation {OpenInterpolation OutsideInput CloseInterpolation}
+ComposedString {Quotes (Interpolation | String)+ Quotes}
+SupportedParsers {JsonPath}
+JsonPath {"json_path '" Identifier "'"}
```

Comment:
should we support the `regex`, `regex_group`, `get_index` and `count` filter here?

Reply:
I'm going to be adding this after, @mathnogueira are we supporting this in the initial release?